packages/flame_lottie/example/lib/main.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flame/game.dart';
+import 'package:flame_lottie/flame_lottie.dart';
+import 'package:flutter/material.dart';
+
+void main() async {
+  runApp(GameWidget(game: LottieExampleGame()));
+}
+
+class LottieExampleGame extends FlameGame {
+  @override
+  Future<void> onLoad() async {
+    final asset = await loadLottie(Lottie.asset('assets/LottieLogo1.json'));
+    add(LottieAnimationComponent(asset));
+  }
+}
+
+class LottieAnimationComponent extends LottieComponent {
```

Comment:
This wrapper component shouldn't be needed, most of the case the user will just use the `LottieComponent` directly, so better have that in the example.